src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  notification.className = `notification ${type}`;
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const titleElement = document.createElement('h2');
+  titleElement.className = 'title';
+  titleElement.textContent = title;
+
+  const descriptionElement = document.createElement('p');
+  descriptionElement.textContent = description;
+
+  notification.appendChild(titleElement);
+  notification.appendChild(descriptionElement);
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.remove();
+  }, 2000);
```

Comment:
The timeout for removing the notification is hardcoded to 2000 milliseconds (2 seconds). Consider making this value configurable by adding an additional parameter to the `pushNotification` function. This would allow different notifications to have different display durations if needed.